core/src/main/java/org/opensearch/sql/executor/pagination/CanPaginateVisitor.java

```diff
@@ -41,19 +41,16 @@
 import org.opensearch.sql.expression.function.BuiltinFunctionName;
 
 /**
- * Use this unresolved plan visitor to check if a plan can be serialized by PaginatedPlanCache.
- * If <pre>plan.accept(new CanPaginateVisitor(...))</pre> returns <em>true</em>,
- * then PaginatedPlanCache.convertToCursor will succeed. Otherwise, it will fail.
- * The purpose of this visitor is to activate legacy engine fallback mechanism.
- * Currently, V2 engine does not support queries with:
- * - aggregation (GROUP BY clause or aggregation functions like min/max)
- * - in memory aggregation (window function)
- * - LIMIT/OFFSET clause(s)
- * - without FROM clause
- * - JOIN
- * - a subquery
- * V2 also requires that the table being queried should be an OpenSearch index.
- * See PaginatedPlanCache.canConvertToCursor for usage.
+ * Use this unresolved plan visitor to check if a plan can be serialized by PaginatedPlanCache. If
+ *
+ * <pre>plan.accept(new CanPaginateVisitor(...))</pre>
+ *
+ * returns <em>true</em>, then PaginatedPlanCache.convertToCursor will succeed. Otherwise, it will
+ * fail. The purpose of this visitor is to activate legacy engine fallback mechanism. Currently, V2
+ * engine does not support queries with: - aggregation (GROUP BY clause or aggregation functions
+ * like min/max) - in memory aggregation (window function) - LIMIT/OFFSET clause(s) - without FROM
+ * clause - JOIN - a subquery V2 also requires that the table being queried should be an OpenSearch
+ * index. See PaginatedPlanCache.canConvertToCursor for usage.
```

Comment:
It looks like you need to add more <pre> tags to this take make all of the lines not lose formatting